README.md

```diff
@@ -130,7 +130,7 @@ performance improvement over query performance. Async iterators needed a lot of
  dumping C* data into HDFS or whatever backup system. A big 60 - 70% gain should be expected.
 
 Phantom connectors now require an ```implicit com.websudos.phantom.connectors.KeySpace``` to be defined. Instead of using a plain string, you just have to
-use ```KeySpace.apply``` or simply: ```trait MyConnector extends Connector { implicit val keySpace = KeySpace("your_def") } ```. This change allows us to
+use ```KeySpace.apply``` or simply: ```trait MyConnector extends CassandraConnector { implicit val keySpace = KeySpace("your_def") } ```. This change allows us to
```

Comment:
Thank you for correcting that. I think we may better go with `SimpleCassandraConnector`, as that will only require defining an implicit keyspace into scope and it's already available in the default imports.